contract-sdk/src/abi/env.rs

```diff
@@ -53,6 +56,17 @@ impl Env for HostEnv {
         // Parse the returned address.
         Address::try_from(dst.as_ref()).unwrap()
     }
+
+    #[cfg(feature = "debug-utils")]
+    fn debug_print(&self, msg: &str) {
+        debug_print(msg)
+    }
+}
+
+#[cfg(feature = "debug-utils")]
+pub(crate) fn debug_print(msg: &str) {
```

Comment:
Why is this separate and `pub(crate)`? It doesn't seem to be used anywhere?

Reply:
It's useful for inserting into the sdk itself before a `Ctx` is available